src/DoctrineMongoODMModule/Options/Configuration.php

```diff
@@ -420,4 +426,20 @@ public function getTypes()
     {
         return $this->types;
     }
+
+    /**
+     * @return string
+     */
+    public function getRepositoryFactory() {
```

Comment:
Apparently we don't have automatic checks here, but the opening curly brace goes in its own line for functions (see PSR-2).